fmc/src/hand_off.rs

```diff
@@ -86,6 +89,27 @@ impl HandOff {
         unsafe { transfer_control(rt_entry) }
     }
 
+    /// Retrieve runtime TCI (digest)
+    pub fn rt_tci(&self, env: &FmcEnv) -> Array4x12 {
```

Comment:
Data Vault has methods for these; could be `env.data_vault().map(|d| d.rt_tci())`, `env.data_vault().map(|d| d.rt_svn())`

Reply:
We need to take the handoff table information into account as it decouples ROM and FMC. We can't hardcode their locations in enums as the data vault driver does.

Reply:
Ah, gotcha.